socorro/unittest/middleware/test_middleware_app.py

```diff
@@ -530,11 +528,20 @@ def _insert_products(self):
         """)
         self.conn.commit()
 
-    def _setup_config_manager(self, extra_value_source=None):
+    def _setup_config_manager(
+        self,
+        extra_value_source=None,
+        services_list=None
+    ):
         if extra_value_source is None:
             extra_value_source = {}
         extra_value_source['web_server.wsgi_server_class'] = MyWSGIServer
         mock_logging = mock.Mock()
+        if services_list:
+            middleware_app.MiddlewareApp.SERVICES_LIST = services_list
+        else:
+            # the global list
+            middleware_app.MiddlewareApp.SERVICES_LIST = middleware_app.SERVICES_LIST
```

Comment:
One belongs to the file, and one belongs to the class. I did that madness so I can get better control over the services list in the tests.